### PR TITLE
Translate to vietnamese

### PR DESCRIPTION
**Pull request by kiennt**
We (ElixirVietnam) finish translating this site to Vietnamese.
Please review this and allow our work to be merged to the main site.

Thanks

### CODE REVIEW

**Comment by doomspork** [comment]
@kiennt this is spectacular!  Allow me to look over things and make sure there aren't any configuration issues and then we can merge this.

Do you and/or ElixirVietnam have a Twitter handle?  I'd love to give you all credit!

File: vi/lessons/basics/pipe-operator.md [hunk 1]
```diff
@@ -1,70 +1,69 @@
 ---
 layout: page
-title: Pipe Operator 
+title: Pipe Operator
```
Comment (doomspork) [inline vi/lessons/basics/pipe-operator.md]: @kiennt there's a few titles that weren't translated like this, was that intentional?

Comment (kiennt) [inline vi/lessons/basics/pipe-operator.md]: @doomspork

Yeah, this is our purpose. Our team discussed while doing the translation, and we decided to not translate some words.

**Comment by kiennt** [comment]
@doomspork 

ElixirVietnam does not have twitter yet, but we have a website http://elixirvn.com. This is just a blog which host in medium, and our team are working to build a homepage to replace it.
I would love if you could mention our site (http://elixirvn.com) :)

BTW, we learned a lot about Elixir while translating ElixirSchool to Vietnamese. Thanks you for a great work.

**Comment by joneslee85** [comment]
@kiennt great work 👏

**Comment by joneslee85** [comment]
@doomspork I'll take care of the review and merger work for you as I understand Vietnamese and in fact I am part of the ElixirVN group

File: vi/lessons/basics/pipe-operator.md (hunk 1 above)
Comment (doomspork) [inline vi/lessons/basics/pipe-operator.md]: Thank you @kiennt!  I just wanted to double check.

**Comment by doomspork** [comment]
Wow @joneslee85 that is a very generous offer, that would be fantastic!  Thank you so very much 😀

**Comment by doomspork** [comment]
Thank you for all your hard work @kiennt!!  Thank you for reviewing and merging @kiennt 💜

**Pull request merged**